scripts/quests/jeuno/In_the_Mood_for_Love.lua

```diff
@@ -0,0 +1,104 @@
+-----------------------------------
+-- In the Mood for Love
+-----------------------------------
+-- Log ID: 3, Quest ID: 69
+-- Odasel  !pos -58 -6 -111
+-- Matoaka !pos -37 -6 -122
+-----------------------------------
+require('scripts/globals/items')
+require('scripts/globals/quests')
+require('scripts/globals/status')
+require("scripts/globals/missions")
+require('scripts/globals/zone')
+require('scripts/globals/interaction/quest')
+-----------------------------------
+local quest = Quest:new(xi.quest.log_id.JEUNO, xi.quest.id.jeuno.IN_THE_MOOD_FOR_LOVE)
+
+quest.reward =
+{
+    gil = 4800,
+    fame = 30,
+    fameArea = xi.quest.fame_area.JEUNO,
+    title = xi.title.PICK_UP_ARTIST,
+}
+
+quest.sections =
+{
+    {
+        check = function(player, status, vars)
+            return status == QUEST_AVAILABLE and
+            player:getCurrentMission(xi.mission.log_id.COP) > xi.mission.id.cop.A_VESSEL_WITHOUT_A_CAPTAIN
+        end,
+
+        [xi.zone.LOWER_JEUNO] =
+        {
+            ['Odasel'] = quest:progressEvent(10035, 0, xi.items.CHAMELEON_DIAMOND),
```

Comment:
Pointed for event params if you don't need other numbers can also be written like
quest:progressEvent(10035, {[1] = xi.items.CHAMELEON_DIAMOND})

Paramas start at 0